src/Text2Image/T2IParamTypes.cs

```diff
@@ -628,6 +628,9 @@ static List<string> listVaes(Session s)
         SaveSegmentMask = Register<bool>(new("Save Segment Mask", "If checked, any usage of '<segment:>' syntax in prompts will save the generated mask in output.",
             "false", IgnoreIf: "false", Group: GroupRegionalPrompting, OrderPriority: 3
             ));
+        SegmentationSortOrder = Register<string>(new("Segmentation Sort Order", "How to sort segments when using '<segment:yolo->' syntax.\nleft-right, top-bottom, bottom-top, largest-smallest, or smallest-largest.\nYou can also use indexes to specify a segment in the given order.\nExmaple: <segment:yolo-face_yolov9c.pt-2> when largest-smallest, will select the second largest face segment.",
+            "left-right", Toggleable: true, IgnoreIf: "left-right", GetValues: _ => ["left-right", "right-left", "top-bottom", "bottom-top", "largest-smallest", "smallest-largest"], Group: GroupRegionalPrompting, OrderPriority: 4
```

Comment:
OrderPriority 4 is identical to `SegmentMaskBlur` - also why this is in between mask params?
... also why does the existing code have overlapping order priorities lol.
Probably move this down to 5.5